alpine/Dockerfile

```diff
@@ -6,33 +6,87 @@ ARG TOOLBOX_EXCLUDED_TAGS="exclude-php:${PHP_VERSION}"
 ARG TOOLBOX_VERSION="1.62.0"
 
 FROM ${BASE_IMAGE} as phpbase
-
 LABEL maintainer="Jakub Zalas <jakub@zalas.pl>"
-ARG BUILD_DEPS="autoconf file g++ gcc libc-dev pkgconf re2c"
 ARG LIB_DEPS="zlib-dev libzip-dev bzip2-dev icu-dev"
 ARG TOOL_DEPS="git graphviz ttf-freefont make unzip gpgme gnupg-dirmngr openssh-client"
+RUN --mount=type=cache,target=/var/cache/apk,sharing=locked,id=apk \
```

Comment:
`sharing=private` here would prevent the `apk` from blocking, but would mean it sometimes wouldn't share the cache (a 2nd or 3rd cache would be created). Not sure which is best